recipe/meta.yaml

```diff
@@ -47,3 +50,4 @@ about:
 extra:
   recipe-maintainers:
     - tadeu
+    - lrineau
```

Comment:
@lrineau is it ok to add you as maintainer?

Reply:
Yes, maybe. For the moment, I do not know conda very well. I will probably only review and give advice. And my colleague @maxGimeno as well.